README.md

```diff
@@ -6,10 +6,11 @@
 [![docs.rs](https://docs.rs/druid/badge.svg)](https://docs.rs/druid/)
 [![license](https://img.shields.io/crates/l/druid)](./LICENSE)
 
-Druid is an experimental Rust-native UI toolkit. Its main goal is to
-demonstrate the potential of Rust as an application programming language, while
-letting users write fast, small, and generally efficient programs with minimal
-hassle.
+Druid is an experimental Rust-native UI toolkit. Its main goal is to offer a
+polished user experience. There are many factors to this goal, including
+performance, a rich palette of interactions (thus a widget library to support
```

Comment:
~thus~ hence ([*Hence usually refers to the future. Thus usually refers to the past.*](https://www.englishgrammar.org/therefore-hence-so-then-thus-etc/))